src/disco/reducers/discoResults.js

```diff
@@ -1,8 +1,13 @@
-import { DISCO_RESULTS } from 'disco/constants';
+import { LOAD_DISCO_RESULTS } from 'disco/constants';
 
 export default function discoResults(state = [], { type, payload }) {
-  if (type === DISCO_RESULTS) {
-    return payload.results;
+  switch (type) {
+    case LOAD_DISCO_RESULTS: {
+      const { entities, result } = payload;
+      // Unwrap the magical discoResults API schema.
```

Comment:
I think the comment is helpful because it's not obvious why the reducer is altering `result.results` in this way. The comment should help someone understand that it's because of the `discoResults` API schema which they can look up in `disco/api.js`.